README.md

```diff
@@ -6,14 +6,14 @@ Discord RPC for VMs.
 
 [![pypresence](https://img.shields.io/badge/using-pypresence-00bb88.svg?style=flat-square&logo=discord&logoWidth=20)](https://github.com/qwertyquerty/pypresence)
 
-**Works with VMware Workstation 14/15, VMware Fusion 14/15 and Hyper-V. VMware Workstation Player 14/15 users, see [here](https://github.com/dhinakg/vm-rpc/blob/master/vix.md).**
+**Works with VMware Workstation 14/15, VMware Fusion 14/15, Hyper-V and VirtualBox 7. VMware Workstation Player 14/15 users, see [here](https://github.com/dhinakg/vm-rpc/blob/master/vix.md).**
 
 ### How to use
 
 1. Create a Discord app. [https://discordapp.com/developers/](https://discordapp.com/developers/)
 2. Install Python 3.
-3. Install `pypresence` from `pip`.
-4. Rename `sample-settings.json` to `settings.json` and configure your settings as neccessary.
+3. Install `pypresence`, `pytz` and `tzlocal` (for vm uptime in VirtualBox) by running `pip -r requirements.txt`.
```

Comment:
Is this really needed? I took a quick look and it seems like VBoxManage returns a UTC time. I would prefer using dateutil.parser to parse the output from VBoxManage, adding the UTC tz information, and then using the timestamp from that. As a bonus, you don't need the user's timezone

Reply:
I don't understand how you want to add the tz information without having the user's timezone and therefore its utc offset. I need the local time in order to display the correct runtime of vbox in discord

Reply:
`int(dateutil.parser.parse(state[state.find('since')+6:state.find('.')]).replace(tzinfo=datetime.timezone.utc).timestamp())`